### ISSUE TITLE
Scale effecting line of sight values

### ISSUE THREAD

**Issue by LadyAlekto** [janxious/MechResizer#4]
LadyAlekto - heute um 17:41 Uhr
@jo i just realized something with your mech scaler and firing lines, can you add a multiplier that affects the los settings?
jo - heute um 17:41 Uhr
there already is one
for each type of line
LadyAlekto - heute um 17:42 Uhr
i mean the way the game calculates los
"LOSTargetPositions": [
        {
            "x": 0,
            "y": 12,
            "z": 1
        },
        {
            "x": 4,
            "y": 12,
            "z": 0
        },
        {
            "x": -4,
            "y": 12,
            "z": 0
        },
        {
            "x": 3,
            "y": 5,
            "z": 0
        },
        {
            "x": -3,
            "y": 5,
            "z": 0
        }
jo - heute um 17:42 Uhr
I need more information about what you want to see then
it would be easy to change the way the shot quality affects things
LadyAlekto - heute um 17:43 Uhr
basically a toggle to make the mech scale apply to these values in the hopes it fixes some weird los points
the values i gave you are in the chassis the position from which a mech calculates what it can see
jo - heute um 17:44 Uhr
ah I see
you want the resizer to apply to everywhere size is used
that was an issue I had with my godzilla mech
lasers coming out of the toes
LadyAlekto - heute um 17:45 Uhr
^^
that should allow mech calses have a real gameplay effect
NEUE NACHRICHTEN
jo - heute um 17:47 Uhr
Would you mind making an issue on the repo for me
just copy this conversation in :smile:
